### PR TITLE
Add note about when /async-function/{functionName} can be invoked.

### PR DESCRIPTION
**Pull request by elliott-beach**
From prior conversation. 
## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: api-docs/swagger.yml [hunk 1]
```diff
@@ -148,6 +148,9 @@ paths:
   /async-function/{functionName}:
     post:
       summary: 'Invoke a function asynchronously in OpenFaaS'
+      description: >-
+        ### This endpoint requires NATS streaming.
```
Comment (alexellis) [inline api-docs/swagger.yml]: Can we re-word to requires the asynchronous stack. Including the link is great.

**Comment by alexellis** [comment]
Looks like we have a small merge conflict now. Please can you look at this @elliott-beach when  you have time.

**Comment by alexellis** [comment]
@elliott-beach hey.. wanted to ping you about this.

**Comment by elliott-beach** [comment]
@alexellis Should be fixed.

I've been busy lately, hopefully will have more time soon.

**Pull request merged**